lib/resources/Authentication.js

```diff
@@ -60,4 +60,15 @@ Authentication.prototype.loginWithKey = function(accessKey, secret, tenantName,
 	});
 };
 
+Authentication.prototype.refreshToken = function(token, scope) {
```

Comment:
Remover scope, pois deixará de fazer parte da APIs.
```suggestion
Authentication.prototype.refreshToken = function(token) {
```